prettier.config.js

```diff
@@ -3,6 +3,7 @@ module.exports = {
   printWidth: 140,
```

Comment:
```suggestion
  printWidth: 120,
```

![image](https://user-images.githubusercontent.com/14183168/76068683-de23b480-5f91-11ea-8684-348c92894ff0.png)

Reply:
I'm doing that in a separate PR on Monday probably (120 does seem to be the most popular choice so far), but I just added a comment in the config so it's clear that's one of the proposed changes to be enabled.